xendit-java-lib/src/main/java/com/xendit/XenditClient.java

```diff
@@ -0,0 +1,85 @@
+package com.xendit;
+
+import com.xendit.model.balance.BalanceClient;
+import com.xendit.model.batchDisbursements.BatchDisbursementClient;
+import com.xendit.model.cardlessCredit.CardlessCreditClient;
+import com.xendit.model.creditCard.CreditCardClient;
+import com.xendit.model.customer.CustomerClient;
+import com.xendit.model.directDebit.DirectDebitPaymentClient;
+import com.xendit.model.disbursement.DisbursementClient;
+import com.xendit.model.ewallet.EWalletClient;
+import com.xendit.model.invoice.InvoiceClient;
+import com.xendit.model.payout.PayoutClient;
+import com.xendit.model.qrCode.QRCodeClient;
+import com.xendit.model.recurringPayment.RecurringPaymentClient;
+import com.xendit.model.retailOutlet.RetailOutletClient;
+import com.xendit.model.virtualAccount.FixedVirtualAccountClient;
+
+public class XenditClient {
+
+  private static Xendit.Option opt;
+  public InvoiceClient invoice;
+  public BalanceClient balance;
+  public PayoutClient payout;
+  public DisbursementClient disbursement;
+  public EWalletClient eWallet;
+  public QRCodeClient qrCode;
+  public CustomerClient customer;
+  public RetailOutletClient retailOutlet;
+  public BatchDisbursementClient batchDisbursement;
+  public CardlessCreditClient cardlessCredit;
+  public CreditCardClient creditCard;
+  public FixedVirtualAccountClient fixedVirtualAccount;
+  public RecurringPaymentClient recurringPayment;
+  public DirectDebitPaymentClient directDebitPayment;
+
+  private XenditClient() {}
+
+  public static class Builder {
+
+    private String apikey;
+
+    public Builder() {}
+
+    public Builder apikey(String apikey) {
```

Comment:
Name this `setApikey` instead?

Reply:
Yes @stanleynguyen, I will change variable naming.

Reply:
> " Btw I wonder if the breaking package path change is really necessary? "

We have created seperate folder for each operation and its client in model package. Instead we can place all operation and its client directly in model package that would not break package path.

Reply:
> Name this `setApikey` instead?

Yes. That can also be change.

Reply:
> Also, we seem to be missing documentation updates?

Yes. That is pending. We will update it.

Reply:
> We have created seperate folder for each operation and its client in model package. Instead we can place all operation and its client directly in model package that would not break package path.

Let's do this instead so we don't have to bump major version?